docs/core/entity.md

```diff
@@ -149,6 +149,11 @@ entity class for details.
 If an integration needs to access its own properties it should access the property (`self.name`), not the class or instance attribute (`self._attr_name`).
 :::
 
+:::tip
+Whenever feasible, prefer the `_attr_` attributes implementation, avoiding dummy proxy getters/setters. This reduces the lines of 
```

Comment:
We actually prefer the use of `EntityDescription`s over entity class or instance attributes.